src/modules/ce/components/ReferralStep.tsx

```diff
@@ -97,51 +100,90 @@ const ReferralStep: React.FC<Props> = ({}) => {
   if (!step || !formDefinition) return <NotFound />;
 
   return (
-    <CommonCard title={name}>
-      <Stack gap={2}>
-        <ReferralStepAssignee step={step} />
-        <Divider />
-        {status === CeReferralStepStatus.Available && (
-          <StartCeReferralStepButton
-            step={step}
-            opportunityId={opportunityId}
-            projectId={projectId}
-            referralId={referralId}
-          >
-            Start Step
-          </StartCeReferralStepButton>
-        )}
-        {status === CeReferralStepStatus.InProgress && (
-          <DynamicForm
-            definition={formDefinition.definition}
-            onSubmit={({ valuesByLinkId, confirmed }) => {
-              submit({
-                variables: {
-                  referralId: referralId,
-                  stepId: stepId,
-                  input: valuesByLinkId,
-                  formDefinitionId: formDefinition.id,
-                  confirmed,
-                },
-              });
-            }}
-            errors={errors}
-            loading={submitLoading}
-            FormActionProps={{
-              submitButtonText: 'Submit',
-              noDiscard: true,
-            }}
-            initialValues={formState}
-          />
-        )}
-        {status === CeReferralStepStatus.Completed && formState && (
-          <DynamicView
-            definition={formDefinition.definition}
-            values={formState}
-          />
-        )}
-      </Stack>
-    </CommonCard>
+    <Stack direction='column' gap={2}>
+      <Paper
+        sx={{ p: 2, justifyContent: 'flex-start' }}
+        component={ButtonLink}
+        startIcon={<BackIcon />}
+        to={generateSafePath(ProjectDashboardRoutes.REFERRAL_STEPS, {
+          projectId,
+          opportunityId,
+          referralId,
+        })}
+      >
+        Back to All Tasks
+      </Paper>
+      <CommonCard title={name}>
+        <Stack gap={2}>
+          <ReferralStepAssignee step={step} />
+          <Divider />
+          {/* TODO - discuss this "Start step" button. Is this status distinction important to the user? */}
```

Comment:
decision:
- when people navigate into the referral, they will go to the overarching referral page
- navigating directly to unstarted step isn't really supported. Return not found